packages/wallet-service/src/schemas.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import Joi from 'joi';
+import { FullNodeApiVersionResponse, EnvironmentConfig } from '@src/types';
+
+export const FullnodeVersionSchema = Joi.object<FullNodeApiVersionResponse>({
```

Comment:
Will it break the code in case we change anything in the full node `/version` API?

Reply:
The `}).unknown(true);` at the end should allow for new keys to be added without breaking the validation.